File: module1/first_call.py
import anthropic

client = anthropic.Anthropic()

response = client.messages.create(
    model="claude-sonnet-4-6",
    max_tokens=1024,
    messages=[
        {"role": "user", "content": "As an API, All things you can handel in my GIT account"}
    ]
)

print(response.content[0].text)
print("\n--- Metadata ---")
print("Input tokens :", response.usage.input_tokens)
print("Output tokens:", response.usage.output_tokens)
print("Stop reason  :", response.stop_reason)

File: module1/first_call.mjs
import Anthropic from "@anthropic-ai/sdk";

const client = new Anthropic();

const response = await client.messages.create({
  model: "claude-sonnet-4-6",
  max_tokens: 1024,
  messages: [
    { role: "user", content: "Say hello and tell me 3 things you can do as an API." }
  ]
});

console.log(response.content[0].text);
console.log("\n--- Metadata ---");
console.log("Input tokens :", response.usage.input_tokens);
console.log("Output tokens:", response.usage.output_tokens);
console.log("Stop reason  :", response.stop_reason);